maps/defficiency.dmm

```diff
@@ -271502,17 +272997,17 @@ aaa
 aaa
 aaa
 aaa
-aaa
-aaa
-aaa
-bwo
-acf
-acf
-aaa
-aaa
-aaa
-aaa
-aaa
+bxy
+aDV
+aHB
+bxu
+hSQ
+hSQ
+hSQ
```

Comment:
> three right particle emitters

You've made it impossible to set up the singularity by deleting the center and left emitters

Reply:
Fixed, this and tested setup engine to make sure it works.